.env.example

```diff
@@ -1,5 +1,7 @@
 VITE_POLYGON_MIGRATION_TRANSACTION_FEE=0.05
+VITE_POLYGON_CONTRACT_ADDRESS=0xContractAddress
+VITE_POLYGON_RPC_URL=https://rpc-mumbai.maticvigil.com/
+
```

Comment:
we'll need a variable for defining the polygonscan url as well, but will make that into another task so we can get this merged early on